src/NServiceBus.AcceptanceTests/Performance/When_message_is_audited.cs

```diff
@@ -30,6 +30,26 @@ public class When_message_is_audited : NServiceBusAcceptanceTest
             Assert.IsTrue(context.IsMessageHandledByTheAuditEndpoint);
         }
 
+        [Test]
+        public async Task Should_contain_proper_time_sent()
```

Comment:
Yeah it does or I can't think of what kind of when should cover the 2nd case. We could also merge both tests as the logic is identical only asserts are different, but then We would have a lot of points of failure.